package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@treehouses/cli",
-  "version": "1.23.29",
+  "version": "1.23.30",
```

Comment:
You don't need to update the version yourself as it might will be a merging conflict when others create other PRs